### PR TITLE
update to Vue 3

### PR DESCRIPTION
**Pull request by cgrayson**
## Description of the change

This updates the RUI to Vue 3. It also replaces the abstract example on `Page2` with the typical "stop or continue on failure" configuration that's often needed.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change
- [ ] Technical Debt
- [ ] Documentation

## Related tickets

https://app.shortcut.com/active-prospect/story/80032/update-integration-template-repo-to-vue-3

### Development and Testing

- [ ]  Lint rules pass locally.
- [ ]  The code changed/added as part of this pull request has been covered with tests, or this PR does not alter production code.
- [ ]  All tests related to the changed code pass in development, or tests are not applicable.

### Code Review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [ ]  At least two engineers have been added as "Reviewers" on the pull request.
- [ ]  Changes have been reviewed by at least two other engineers who did not write the code.
- [ ]  This branch has been rebased off master to be current.

### Tracking 
- [ ]  Issue from Shortcut/Jira has a link to this pull request.
- [ ]  This PR has a link to the issue in Shortcut.

### QA
- [ ]  This branch has been deployed to staging and tested.

### CODE REVIEW

File: lib/ui/public/app/store.js [hunk 1]
```diff
@@ -57,7 +50,7 @@ const initStore = (config, ui) => new Vuex.Store({
             id: config.entity.id
           },
           integration: {
-            module_id: config.integration || 'leadconduit-service_being_integrated.outbound.handle',
```
Comment (jgrayson676) [inline lib/ui/public/app/store.js:60]: I don't feel super strongly about this, but I'd think that leaving `config.integration` in the template is fine as that's the pattern most add-ons with more than one `module_id` use. But if you'd rather it be something that should be intentionally updated after copying over the template that's fair too and fine by me.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,37 +0,0 @@
-## Description of the change
```
Comment (jgrayson676) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: I think it's fine to remove these, but I wonder if they won't just get recreated by Kelly's script the next time that runs and/or is updated.

**Review by jgrayson676 — APPROVED** [review]
otherwise LGTM

File: lib/ui/public/app/store.js (hunk 1 above)
Comment (cgrayson) [reply lib/ui/public/app/store.js]: Good point, I wasn't thinking of the multiple add-on case. Reverted.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (cgrayson) [reply .github/PULL_REQUEST_TEMPLATE.md]: 🤦 Ha ha, you're right. I was thinking to keep them from being included in new repos, but that will have to be a change to `init.sh` instead. Reverted these, too.

**Review by jgrayson676 — APPROVED** [review]

**Pull request merged**